crawl-ref/source/dat/descript/spells.txt

```diff
@@ -179,7 +179,8 @@ Fires a penetrating bolt of flames.
 %%%%
 Bolt of Light spell
 
-Fires a ray of searing light, injuring and blinding those struck.
+Fires a ray of searing light, injuring and blinding those struck. Confuses
+instead of blinding if the player is the victim.
```

Comment:
I don't think we ever want to use 'the player' in a description as it breaks immersion. Perhaps a separate paragraph saying something like "If the bolt hits you, you would be confused' would better?

Otherwise I'm excited for this PR as currently monster wand damage is very obscure.

Reply:
Sounds reasonable as "the player" isn't mentioned in any other spell descriptions.